### DISCUSSION TITLE
Unified vite/webpack analyze product

### DISCUSSION THREAD

**Discussion by nonzzz** [umijs/umi#12456] [Ideas]
I notice in vite mode. umi is using `rollup-plugin-visualizer` to display the bundle analyzer result. But isn't same with webpack version. AFAIK i think the `rollup-plugin-visualizer` squarifed treemap isn't convenient to operate. So i made a new plugin [vite-bundle-analyzer](https://www.npmjs.com/package/vite-bundle-analyzer) 

## Why new plugin?

- `vite-bundle-analyzer` is focus on the behavior with webpack version and has a very small bundle size and faster loading bundle result speed.

And it also has a certain number of users. Here are the users list. 

<img width="962" alt="image" src="https://github.com/umijs/umi/assets/52351095/807c7b85-051d-4fa3-ba38-ff3dcb2835ef">

**Comment by fz6m** [comment]
@xierenyuan 可以看下这个插件如何，如果更好有帮助的话，可以考虑把 `rollup-plugin-visualizer` 给替换掉，这个应该只是小插件，替换了没啥影响。

另外我理解用户可以自己用 `modifyViteConfig` 来添加这个插件手动使用。
